src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/mirrorpoint/create/actions/ResourceAllocationAction.java

```diff
@@ -53,33 +53,53 @@
 
 @Slf4j
 public class ResourceAllocationAction
-        extends BaseResourceAllocationAction<FlowMirrorPointCreateFsm, State, Event, FlowMirrorPointCreateContext> {
+        extends NbTrackableWithHistorySupportAction<FlowMirrorPointCreateFsm, State, Event,
+        FlowMirrorPointCreateContext> {
 
+    private final int pathAllocationRetriesLimit; // Will be used in long mirrors
+    private final int pathAllocationRetryDelay; // Will be used in long mirrors
+    private final int resourceAllocationRetriesLimit; // Will be used in long mirrors
     private final FlowMirrorPointsRepository flowMirrorPointsRepository;
     private final FlowMirrorPathRepository flowMirrorPathRepository;
     private final KildaConfigurationRepository kildaConfigurationRepository;
+    private final FlowResourcesManager resourcesManager;
+    private final PathComputer pathComputer; // Will be used in long mirrors
```

Comment:
if those are going to be used only in long mirrors, maybe you can somehow add it to the variable names?

Reply:
I just added this comments to point why this fields are unused now. When long mirrors will be implemented I will remove thee comments.